compliance/nvidia/TEST04/verify_performance.py

```diff
@@ -52,13 +52,11 @@ def main():
         if ref_mode == "SingleStream":
             if re.match(".*Early stopping 90th percentile estimate", line):
                 ref_score = line.split(": ",1)[1].strip()
-                ref_score = 1e9 / float(ref_score)
```

Comment:
This change can potentially break the below check as the check assumes throughput metric for all the scenarios. 
https://github.com/mlcommons/inference/pull/1491/files#diff-5c101fd75c9062a7dec72722d5f4aafe66e7d55fbc32f97378b74c08034c272cR141

Reply:
TEST01 is doing comparison like [this](https://github.com/mlcommons/inference/blob/master/compliance/nvidia/TEST01/verify_performance.py#L142) - both upper and lower bounds are checked and so we don't need reciprocal but the check is stricter than required. 

TEST05 also had a similar check but was removed in a PR before 3.1 and so is currently broken for the offline scenario. [This PR](https://github.com/mlcommons/inference/pull/1463) should fix that.

Reply:
Since the TEST05-related PR has been merged, should we do TEST04 in the same way?